src/ui/public/angular-bootstrap/datepicker/month.html

```diff
@@ -1,15 +1,72 @@
-<table role="grid" aria-labelledby="{{uniqueId}}-title" aria-activedescendant="{{activeDateId}}">
+<table
+  class="kuiDatePicker"
+  role="grid"
+  aria-labelledby="{{uniqueId}}-title"
+  aria-activedescendant="{{activeDateId}}"
+>
   <thead>
     <tr>
-      <th><button type="button" class="btn btn-default btn-sm pull-left" ng-click="move(-1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-left"></i></button></th>
-      <th><button id="{{uniqueId}}-title" role="heading" aria-live="assertive" aria-atomic="true" type="button" class="btn btn-default btn-sm" ng-click="toggleMode()" tabindex="-1" style="width:100%;"><strong>{{title}}</strong></button></th>
-      <th><button type="button" class="btn btn-default btn-sm pull-right" ng-click="move(1)" tabindex="-1"><i class="glyphicon glyphicon-chevron-right"></i></button></th>
+      <th
+        class="kuiDatePickerNavigationCell"
+        colspan="3"
+      >
+        <div class="kuiDatePickerNavigation">
+          <button
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="move(-1)"
+            tabindex="-1"
+          >
+            <span class="kuiIcon fa-chevron-left"></span>
+          </button>
+
+          <button
+            id="{{uniqueId}}-title"
+            role="heading"
+            aria-live="assertive"
+            aria-atomic="true"
+            type="button"
+            class="kuiDatePickerNavigationButton"
+            ng-click="toggleMode()"
+            tabindex="-1"
+            style="width:100%;"
```

Comment:
Are we comfortable with this inline-style even after the conversion to the ui-framework?